src/test/shared/sam/cli/samCliLocalInvoke.test.ts

```diff
@@ -0,0 +1,329 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+import * as assert from 'assert'
+import * as del from 'del'
+import * as path from 'path'
+import * as vscode from 'vscode'
+import { writeFile } from '../../../../shared/filesystem'
+import { mkdtemp } from '../../../../shared/filesystemUtilities'
+import { TestLogger } from '../../../../shared/loggerUtils'
+import { DefaultSamCliTaskInvoker } from '../../../../shared/sam/cli/samCliInvoker'
+import { SamCliLocalInvokeInvocation } from '../../../../shared/sam/cli/samCliLocalInvoke'
+
+describe('SamCliLocalInvokeInvocation', async () => {
+
+    class FakeTaskExecution implements vscode.TaskExecution {
+        public constructor(public readonly task: vscode.Task) {
+        }
+
+        public terminate(): void {
+            throw new Error('Method not implemented.')
+        }
+    }
+
+    class TestTaskInvoker implements DefaultSamCliTaskInvoker {
```

Comment:
Should this be `implements SamCliTaskInvoker`?

Reply:
Whoa! Good catch